src/status_im/ui2/screens/chat/composer/view.cljs

```diff
@@ -170,7 +170,7 @@
                                     (re-frame/dispatch [:dismiss-keyboard])) edit)
               [reanimated/view {:style (reanimated/apply-animations-to-style
                                         {:height shared-height}
-                                        {})}
+                                        {:z-index 2})}
```

Comment:
z-index 2, so that chat composer overlap floating shell button

Reply:
> > any updates to this?
> 
> hi @du64, Implementation is still in progress and needs a few bug fixes. And also I was hoping to extend this functionality for group chats and communities too in the same PR, so will take some time.

This p.r is already considerably large. Is it possible to add group chats and communities in follow up p.rs? 🤔

Reply:
Is this an iOS only thing, Should we add elevation ?

Reply:
https://reactnative.dev/docs/layout-props#zindex
> zIndex controls which components display on top of others.

Its not for elevation or shadow, it only controls order of view in document tree